templates/guides/version-2-upgrade.html.md

```diff
@@ -0,0 +1,221 @@
+---
+subcategory: ""
+page_title: "Terraform Sym Provider Version 2 Upgrade Guide"
+description: |-
+  Terraform Sym Provider Version 2 Upgrade Guide
+---
+
+# Terraform Sym Provider Version 2 Upgrade Guide
+
+Version 2.0.0 of the Sym provider for Terraform is a major release and includes some changes that you will need to consider when upgrading. This guide is intended to help with that process.
+
+The full list of changes can always be found in the [Terraform Sym Provider Releases](https://github.com/symopsio/terraform-provider-sym/releases).
+
+Upgrade topics:
+
+<!-- TOC depthFrom:2 depthTo:2 -->
+
+- [Terraform Sym Provider Version 2 Upgrade Guide](#terraform-sym-provider-version-2-upgrade-guide)
+  - [Provider Version Configuration](#provider-version-configuration)
+  - [Resource: sym_flow](#resource-sym_flow)
+    - [`template` has been removed](#template-has-been-removed)
+    - [`params` is now a nested argument](#params-is-now-a-nested-argument)
+    - [`prompt_fields_json` has been removed](#prompt_fields_json-has-been-removed)
+    - [`allowed_sources_json` has been removed](#allowed_sources_json-has-been-removed)
+    - [`list` has been removed as a valid type for `prompt_field`.](#list-has-been-removed-as-a-valid-type-for-prompt_field)
+
+<!-- /TOC -->
+
+## Provider Version Configuration
+
+-> Before upgrading to version 2.0.0 or later, it is recommended to upgrade to the most recent 1.X version of the provider (version 1.14.1) and ensure that your environment successfully runs [`terraform plan`](https://www.terraform.io/docs/commands/plan.html) without unexpected changes or deprecation notices.
```

Comment:
Does the `->` make an info block?

Reply:
Yes! That seems to be what it's done in AWS's page